components/automate-ui/src/app/modules/infra-proxy/client-details/client-details.component.scss

```
@@ -96,27 +96,44 @@ img {
     @include base-box-shadow();
     background: $chef-white;
     border-radius: $global-radius;
+    border: 0.5px solid $chef-key-tab;
   }
 
   .key-tab-body {
     display: flex;
     padding: 1.5em;
     background: $chef-white;
-    border: 0.5px solid $chef-key-tab;
+
+    chef-snippet {
+      width: 100%;
+    }
+
   }
 
   .key-tab-body-top {
-    padding: 1em;
+    margin: 0 10px;
     background: $chef-white;
-    border-left: 0.5px solid $chef-key-tab;
-    border-right: 0.5px solid $chef-key-tab;
-    border-top: 0.5px solid $chef-key-tab;
+    border-bottom: 0.5px solid $chef-key-tab;
+    text-align: right;
 
-    p {
-      margin: 0;
-      text-align: right;
-      color: $chef-dark-blue;
+    chef-button:focus {
+      outline: 0.5px solid $chef-primary-bright;
     }
+
+    ::ng-deep {
+      chef-button button:focus {
+        outline: 0.5px solid $chef-primary-bright;
```

```suggestion
        outline: -webkit-focus-ring-color auto 1px;
```
This will give us our default focus ring

---

@SEAjamieD As per UX suggestions, added outline with different colour and different width. Focus changed to 0.5px solid #3864F2

---

according to your comment below, this may be able to stay as is, since now UX is requesting 0.5px 👍